jsk_apc2016_common/config/bin_lower_shelf.yaml

```diff
@@ -1,7 +1,7 @@
-initial_pos_list: [[-0.27, 0.25, -0.41], [-0.24, -0.01, -0.41], [-0.24, -0.32, -0.41], [-0.27, 0.25, -0.12], [-0.24, -0.01, -0.12], [-0.24, -0.32, -0.12]]
+initial_pos_list: [[-0.22, 0.280, -0.116], [-0.22, 0, -0.116], [-0.22, -0.280, -0.116], [-0.22, 0.280, -0.361], [-0.22, 0, -0.361], [-0.22, -0.280, -0.361]]
 initial_rot_list: [[0,0,0], [0,0,0], [0,0,0], [0,0,0], [0,0,0], [0,0,0]]
 initial_quat_list: [[0,0,0,1], [0,0,0,1], [0,0,0,1], [0,0,0,1], [0,0,0,1], [0,0,0,1]]
-dimensions: [[0.38, 0.23, 0.20],[0.38, 0.27, 0.20],[0.38, 0.23, 0.20], [0.38, 0.23, 0.18],[0.38, 0.27, 0.18],[0.38, 0.23, 0.18]]
+dimensions: [[0.37, 0.248, 0.192],[0.37, 0.306, 0.192],[0.37, 0.248, 0.192], [0.37, 0.248, 0.218],[0.37, 0.306, 0.218],[0.37, 0.248, 0.218]]
 frame_id_list: [kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base]
-prefixes: [bin_j, bin_k, bin_l, bin_g, bin_h, bin_i]
+prefixes: [bin_g, bin_h, bin_i, bin_j, bin_k, bin_l]
```

Comment:
Don't change the order of prefixes.
If you do this, the correspondence of bin to bin name is changed.